Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp

```diff
@@ -797,9 +819,10 @@ GRefPtr<GSubprocess> bubblewrapSpawn(GSubprocessLauncher* launcher, const Proces
         static std::unique_ptr<XDGDBusProxy> dbusProxy = makeUnique<XDGDBusProxy>();
         if (dbusProxy)
             bindDBusSession(sandboxArgs, *dbusProxy, flatpakInfoFd != -1);
-        // FIXME: We should move to Pipewire as soon as viable, Pulse doesn't restrict clients atm.
+        // FIXME: Disable PulseAudio access when Pipewire is available.
```

Comment:
So since we don't have to allow compat with arbitrary applications, I think we can actually just go ahead and do this now? As long as GStreamer and WebKit do not need PulseAudio anymore, we should be good?